lib/kubeClass.js

```diff
@@ -119,6 +119,15 @@ module.exports = class KubeClass {
   }
 
 
+  injectSelfLink(object, resourceMeta) {
+    let resources = objectPath.get(object, 'items', []);
```

Comment:
Is it possible to get an individual resource here instead if a list?

Reply:
the way its making the request `let response = await request.get(resourceMeta.uri(), options);` tells me it is always doing a listing, never getting a specific resource. eg. `resourceMeta.uri()` doesnt specify a name or namespace in the options, so it will always be a list of resources